compatibility-certification-request.md

```diff
@@ -0,0 +1,20 @@
+- [ ] Statement of Acceptance of the terms of the EFTL
+  // Add here
```

Comment:
Would something like this be better?

- [ ] By checking this box I acknowledge that I accept the terms of the [EFTL](https://www.eclipse.org/legal/tck.php).